civicrm_resources/civicrm_resources.module

```diff
@@ -213,3 +51,25 @@ function civicrm_resources_load($extention_full_name, $files = []) {
 function civicrm_resources_flush_caches() {
   return ['cache_civicrm_resources'];
 }
+
+/**
+ * Includes CSS/JS resources onto a page
+ *
+ * @param string $extension_path
+ *   Machine path of civicrm extension.
+ * @param array $resources_uris
+ *   Resource URI relative to extension paths to be included
+ */
+function _civicrm_resources_add_resource($extension_path, $resource_uri) {
+  $resource_type = pathinfo($resource_uri)['extension'];
```

Comment:
Trying to access the extension like this will result in a notice if the $resource_uri doesn't contain an extension. To avoid that, you can ask pathinfo to only return the extension. This way, it will return a string instead of an array: `pathinfo($resource_uri, PATHINFO_EXTENSION)` instead

Reply:
done.